src/Symfony/Component/ErrorHandler/DebugClassLoader.php

```diff
@@ -468,8 +469,9 @@ public function checkAnnotations(\ReflectionClass $refl, string $class): array
         self::$finalMethods[$class] = [];
         self::$internalMethods[$class] = [];
         self::$annotatedParameters[$class] = [];
+        self::$finalConstants[$class] = [];
         foreach ($parentAndOwnInterfaces as $use) {
```

Comment:
let's keep a blank line before

Reply:
There is no blank line currently :sweat_smile: